src/components/organisms/checkinSteps/orderDetails/checkBookingInfo/checkBookingInfo.jsx

```diff
@@ -11,6 +12,89 @@ const CheckBookingInfo = ({ reservation }) => {
+            class: reservation.car.class,
+            carImage:
+                'https://user-images.githubusercontent.com/48051912/120997146-42ca5200-c787-11eb-9b01-1a458b0664ed.png',
+            checkedIn: reservation.car.checkedIn,
+            driverOne: {
+                role: reservation.car.driverOne.role,
+                driver: reservation.car.driverOne.driver,
+                method: reservation.car.driverOne.method,
+                verified: reservation.car.driverOne.verified,
+            },
+            otherInfo: {
+                freeKM: reservation.car.otherInfo.freeKM,
+                deposit: reservation.car.otherInfo.deposit,
+                ownRisk: reservation.car.otherInfo.ownRisk,
+                priceExtraKM: reservation.car.otherInfo.priceExtraKM,
+            },
+            rentPrice: reservation.car.rentPrice,
+            handInDate: reservation.car.handInDate,
+            handInTime: reservation.car.handInTime,
+            pickUpDate: reservation.car.pickUpDate,
+            pickUpTime: reservation.car.pickUpTime,
+            extraDriver: reservation.car.extraDriver,
+            paidDeposit: reservation.car.paidDeposit,
+            lowerOwnRisk: reservation.car.lowerOwnRisk,
+            orderDetails: true,
+            reservationID: reservation.car.reservationID,
+            handInLocation: reservation.car.handInLocation,
+            pickUpLocation: reservation.car.pickUpLocation,
+            walletSerialNumber: reservation.car.walletSerialNumber,
+            verificationProcess: reservation.car.verificationProcess,
+        }
+        if (index === 0) {
+            const { data, error } = await supabase
+                .from('users')
+                .update({ carResOne: newObject })
+                .eq('userID', userID)
+            if (!data) {
+                console.log(error)
+            } else {
+                console.log(data)
+            }
+        } else if (index === 1) {
+            const { data, error } = await supabase
+                .from('users')
+                .update({ carResTwo: newObject })
+                .eq('userID', userID)
+            if (!data) {
+                console.log(error)
+            } else {
+                console.log(data)
+            }
+        } else if (index === 2) {
+            const { data, error } = await supabase
+                .from('users')
+                .update({ carResThree: newObject })
+                .eq('userID', userID)
+            if (!data) {
+                console.log(error)
+            } else {
+                console.log(data)
+            }
+        }
+    }
+
+    async function handleClick() {
+        const resID = reservation.car.reservationID
+        const userID = reservation.user.userID
+        const index = reservation.carkey
+
+        getSpecificUser(resID, userID, index)
+
+        setTimeout(() => {
+            window.location.href = '/verification'
+        }, 100)
+    }
+
```

Comment:
Can you remove this object, and implement it from [db/updateDatabase.js](https://github.com/joordy/car-checkin-tool/blob/main/src/db/updateDatabase.js). If you write a specific function over here, and pass the correct parameters (check how many drivers, etc), it will make the component code much cleaner.

Reply:
Will do right away!